frontend/src/config/constants.js

```diff
@@ -3,64 +3,83 @@ export const PATIENT_TERM_CAP = "Person";
 export const PATIENT_TERM_PLURAL = "people";
 export const PATIENT_TERM_PLURAL_CAP = "People";
 
-export const stateCodes = [
-  "AK",
-  "AL",
-  "AR",
-  "AS",
+export const liveJurisdictions = [
   "AZ",
-  "CA",
   "CO",
-  "CT",
-  "DC",
-  "DE",
   "FL",
-  "FM",
-  "GA",
-  "GU",
```

Comment:
Should we add Guam to the live states? @emmastephenson

Reply:
Yes! Sorry I didn't respond earlier, I can add it now.